controllers/factory.go

```diff
@@ -73,7 +73,7 @@ func (f *Factory) Build(ctx context.Context) (*Reconcilers, error) {
 }
 
 func (f *Factory) WithClusterReconciler(capiProviders []clusterctlv1.Provider) *Factory {
-	f.dependencyFactory.WithGovc()
+	f.dependencyFactory.WithGovc().WithCmkFromSecrets()
```

Comment:
I think this would go in `withCloudstackClusterReconciler` since it's a cloudstack only dependency

Reply:
Sounds good, will remove. I was going off the Govc example.